sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -3504,6 +3504,16 @@ object functions {
     ArrayContains(column.expr, lit(value).expr)
   }
 
+  /**
+   * Returns null if the array is null, true if the array a1 contains all elements in a2,
+   * and false otherwise. Ignore duplicates and element order in array.
+   * @group collection_funcs
+   * @since 3.1.0
```

Comment:
Should be 3.2.0 as you updated other place.

Reply:
> Should be 3.2.0 as you updated other place.

Miss this part, updated